rnd/autogpt_server/autogpt_server/blocks/talking_head.py

```diff
@@ -0,0 +1,145 @@
+import time
+from typing import Literal
+
+import requests
+
+from autogpt_server.data.block import Block, BlockCategory, BlockOutput, BlockSchema
+from autogpt_server.data.model import BlockSecret, SchemaField, SecretField
+
+
+class ClipsBlock(Block):
+    class Input(BlockSchema):
+        api_key: BlockSecret = SecretField(
+            key="did_api_key", description="D-ID API Key"
+        )
+        script_input: str = SchemaField(
+            description="The text input for the script", default="Welcome to AutoGPT"
+        )
+        provider: Literal["microsoft", "elevenlabs", "amazon"] = SchemaField(
+            description="The voice provider to use", default="microsoft"
+        )
+        voice_id: str = SchemaField(
+            description="The voice ID to use", default="en-US-JennyNeural"
+        )
```

Comment:
maybe add a link to the description pointing to the docs w/ a list of available voices or how to find a voice ID?